wger.py

```diff
@@ -318,19 +354,140 @@ def save_ex_details(self,dir_name):
             </html>"""
             f.write(message)
             f.close()
-        return f"Exercise details were saved successfully to {dir_name} directly."
-
-
-
-user1=Api("Token ffb50c5ea4d743aefa7540aa176590d7de57c907","lilijora","belive2021")
-def main_program(user_name):
-    # print(user_name.chose_weight('2021-08-18',48))
-    # print(user_name.week_program(1750,1300))
-    # print(user_name.create_workout('Workout1', 3))
-    print(user_name.save_ex_details('exercise_details'))
-    # print(user_name.schedule("Schedule1","2021-08-18"))
-    # print(user_name.workout_in_schedule('Schedule1','Workout1',4))
-    # print(user_name.delete_exercise('Workout1','Day 2','Jumping Jacks'))
-    # print(user_name.delete_workout('Workout1'))
+        return len(os.listdir(dir_name))>0
+
+
+
+class TomlClass(Api):
```

Comment:
All this code should be put under the Api() class you developed above, in order for you to use the already created methods in order to make the REST Requests based on the TOML files.